website/content/en/docs/provisioner.md

```diff
@@ -148,6 +156,21 @@ spec:
     clusterDNS: ["10.0.1.100"]
 ```
 
+## spec.limits
+
+The provisioner spec includes a limits section (`spec.limits.resources`), which constrains the maximum amount of resources that the provisioner will manage. 
+
+Presently, Karpenter supports `memory` and `cpu` limits. 
+
+CPU limits are described with a `DecimalSI` value, usually a natural integer. 
+
+Memory limits are described with a [`BinarySI` value, such as 1000Gi.](https://github.com/kubernetes/apimachinery/blob/4427f8f31dfbac65d3a044d0168f84c51bfda440/pkg/api/resource/quantity.go#L31)
```

Comment:
We could probably borrow the language from here - https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory 
Same for CPU.

Reply:
Or even link to that page :)

Reply:
I was looking for a page like that to link to!